dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -158,7 +164,15 @@ RtpsUdpDataLink::open(const ACE_SOCK_Dgram& unicast_socket)
     }
   }
 
-  /// FUTURE: Set TTL on the alternate sockets.
+  if (!OpenDDS::DCPS::set_socket_ttl(ipv6_alternate_socket_, ttl)) {
```

Comment:
Use this same helper function everywhere else multicast TTL is set in the codebase (also, rename to have "multicast" in the name?)